submissions/DmytryjK/HTML-CSS-Popup/css/components/fonts.css

```diff
@@ -0,0 +1,14 @@
+@font-face {
+  font-family: "inter-regular";
+  src: url("../fonts/Inter2/converted/Inter-Regular.eot");
```

Comment:
For modern browsers enough `.woff` and `woff2` fonts.

Reply:
fixed